pandas/tests/extension/base/ops.py

```diff
@@ -124,7 +124,7 @@ def test_direct_arith_with_ndframe_returns_not_implemented(self, data, box):
             result = data.__add__(other)
             assert result is NotImplemented
         else:
-            raise pytest.skip(f"{type(data).__name__} does not implement add")
+            pytest.skip(f"{type(data).__name__} does not implement add")
```

Comment:
also not clear here why its a skip instead of xfail

Reply:
Yeah same. Changed to xfail